composer.json

```diff
@@ -80,16 +80,21 @@
         "egulias/email-validator": "~1.2"
     },
     "autoload": {
-        "psr-0": { "Symfony\\": "src/" },
+        "psr-0": {
+            "Symfony\\Bridge\\Doctrine\\": "src/",
+            "Symfony\\Bridge\\Monolog\\": "src/",
+            "Symfony\\Bridge\\ProxyManager\\": "src/",
+            "Symfony\\Bridge\\Swiftmailer\\": "src/",
+            "Symfony\\Bridge\\Twig\\": "src/",
+            "Symfony\\Bundle\\": "src/",
+            "Symfony\\Component\\": "src/"
```

Comment:
It only adds a few entries in the namespace map, and at most 6 call to `strpos`, nothing more.
On the bonus side, it will remove one `file_exists check` when `Symfony\Cmf` is used, so the net benefit is positive.